extensions/README.md

```diff
@@ -1,22 +1,33 @@
-# Extensions
+# Content Extensions
```

Comment:
Not sure whether this are all "content" extensions, e.g. collection assets or item assets seem to be more on the "behavior" side and don't add much "additional" content...

Also, it sounds like there are other extensions for STAC spec, but where are they? So I guess I'd just replace content extension with extension throughout the whole document.

Reply:
"Content" was to distinguish it from "API" -- but I think it should be obvious by it being in this spec rather than the api spec. I'll remove it. 

I also have some clarifying text for the Extensions to clarify if the fields apply to Properties, Assets, or both.  I wanted to clarify this text because one of my co-workers recently started working with the Label Extension, and mistakenly thought the fields were supposed to be added to the Item rather than Item Properties.

Reply:
Agree that it's all 'content' extensions now in the core spec. 

And clarifying text on extensions applying to properties, assets or both is definitely appreciated. With the ability to put things at either level it's good to tell people where they make more sense, and the special situations that might warrant both or breaking from the norm.

Reply:
Added #972 to track this.